taichi/program/program.cpp

```diff
@@ -274,9 +274,6 @@ void Program::initialize_runtime_system(StructCompiler *scomp) {
                                   (void *)&KernelProfilerBase::profiler_start);
     runtime->call<void *, void *>("LLVMRuntime_set_profiler_stop", llvm_runtime,
                                   (void *)&KernelProfilerBase::profiler_stop);
-  } else {
-    runtime->call<void *, void *>("LLVMRuntime_set_profiler", llvm_runtime,
-                                  nullptr);
```

Comment:
Would you tell me why removing this make things work?

Reply:
It's just because on certain platforms `LLVMRuntime_set_profiler` is not found. Maybe due to name mangling. I haven't got a chance to figure it out.